html/janus.js

```diff
@@ -988,7 +990,20 @@ function Janus(gatewayCallbacks) {
 				ws.close();
 				ws = null;
 			} else {
-				navigator.sendBeacon(server + "/" + sessionId, JSON.stringify(request));
+        var uri = server + "/" + sessionId;
+        if (unloadParams) {
+          Object.keys(unloadParams)
+            .filter(function(key) { return unloadParams[key] != null; }) // Ignore null or undefined params
+            .forEach(function(key) {
+              // If there are more than 1 param, we need to use '&' instead of '?' after first param
+              if (uri.search(/\?.*=.*/) >= 0) {
```

Comment:
I will try to fix that to search everything after server + session id and search for "?<key>=<value>" to know if there is already a param with a value as first parameter. Thanks for pointing out that it could search for a question mark in any part of the uri.

I'll try to fix it ASAP.